File: Java/1-Important-Programs/06-greatest-of-three-numbers.java
import java.util.Scanner;

class Main {
    public static void main(String[] args) {
        Scanner getInput = new Scanner(System.in);

        System.out.println("Enter three numbers:");
        int num1 = getInput.nextInt();
        int num2 = getInput.nextInt();
        int num3 = getInput.nextInt();

        if ((num1 > num2) && (num1 > num3)) {
            System.out.println("\nGreatest number is: " + num1);
        }
        else if ((num2 > num1) && (num2 > num3)) {
            System.out.println("\nGreatest number is: " + num2);
        }
        else if ((num3 > num1) && (num3 > num2)){
            System.out.println("\nGreatest number is: " + num3);
        }
        else {
            System.out.println("\nInvalid Inputs.");
        }

        getInput.close();
    }
}

File: Python/1-Important-Programs/06-greatest-of-three-numbers.py
num1 = int(input("Enter first number: "))
num2 = int(input("Enter second number: "))
num3 = int(input("Enter third number: "))

if (num1 == num2 == num3):
    print("\nAll are equal.")
elif (num1 >= num2) and (num1 >= num3):
    print("\nGreatest Number:", num1)
elif(num2 >= num1) and (num2 >= num3):
    print("\nGreatest Number:", num2)
else:
    print("\nGreatest Number:", num3)


# Using pre-built Function
print(max(num1, num2, num3))